unittest/test_inertial_parameters.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+
+###############################################################################
+# BSD 3-Clause License
+#
+# Copyright (C) 2023-2024, Heriot-Watt University
+# Copyright note valid unless otherwise stated in individual files.
+# All rights reserved.
+###############################################################################
+
+import os
+import random
+import time
+import unittest
+
+import numpy as np
+import pinocchio
+
+import rospy
+import rosunit
+
```

Comment:
Support ROS 2 as done in https://github.com/RobotMotorIntelligence/crocoddyl_msgs/blob/devel/unittest/test_whole_body_state.py#L19-L24